### ISSUE TITLE
type Variable doesn't have round

### ISSUE THREAD

**Issue by wangyirui** [facebookresearch/fairseq-py#105]
I am running the fairseq-py on PyTorch 0.4.0, the program report "type Variable doesn't have round" (which occurs in train.py around line 270 when computing loss). I replace the round with torch.round and then it works.

**Comment by edunov** [comment]
Seems the same as #102 
And is likely related to these changes in pytorch: https://github.com/pytorch/pytorch/wiki/Breaking-Changes-from-Variable-and-Tensor-merge 

I'll prepare a fix

**Comment by edunov** [comment]
The issue should be fixed in the latest version of fairseq-py: #107
